package.json

```diff
@@ -31,28 +31,29 @@
     "main": "can-stache-bindings"
   },
   "dependencies": {
+    "can-attribute-encoder": "^0.3.0",
```

Comment:
what do you mean?  It should be hard-coded because the API might change.  Everything included by core shouldn't break if it gets updated.